packages/stk/stk_ngp/stk_ngp/NgpReductions.hpp

```diff
@@ -74,14 +74,7 @@ struct ReductionTeamFunctor
         JoinOp (value_type& value_) : value (&value_) {}
 
```

Comment:
I'm not sure if you're suppose to keep a pointer to the value inside the reduction functor.  The result is just the output of team `parallel_reduce`, right?